classes/Search.php

```diff
@@ -132,7 +132,20 @@ class SearchCore
 
     public static function extractKeyWords($string, $id_lang, $indexation = false, $iso_code = false)
     {
-        $sanitizedString = Search::sanitize($string, $id_lang, $indexation, $iso_code, false);
+        if (!stripos($string, '&')) {
+            $sanitizedString = Search::sanitize($string, $id_lang, false, Context::getContext()->language->iso_code);
```

Comment:
Why are you using context's iso_code instead of the one from this functions' argument?

Reply:
Hmm... valid point, partially out of habit I suppose; but wouldn't be better to enforce the iso_code? 
This variable is supposed to be a string 2-letter iso code as far as I know.
Is there a scenario where at this point the iso_code can be false?

Reply:
@Rho-bur if we enforce it, it will be a BC Break. From what I see, `$iso_code` seems to be used every time we call `extractKeyWords` so I guess you can rely on it instead of using `Context::getContext()->language->iso_code` ;)

Reply:
@atomiix alright, following your recommendation, I have commit'ed the change.